packages/migrate/src/__tests__/DbPull.test.ts

```diff
@@ -480,6 +481,947 @@ describe('postgresql - missing database', () => {
   })
 })
 
+describe('postgresql views re-introspection warnings', () => {
```

Comment:
It's unexpected/surprising that these come before plain Introspection to me.

Reply:
This is what @jkomyno recommended when we paired on this 😅 Didn't think to ask about it at the time

Reply:
Maybe doesn't matter - it just took me a bit of scrolling to understand that the first big block is Re-Introspection (with existing schema file), and the second one is normal Introspection.